tools/packaging/common/sidecar.env

```diff
@@ -102,3 +102,8 @@ OUTPUT_CERTS=/etc/certs
 # and returning workload certificates. Istiod is implementing the protocol, and is the default value
 # if ISTIO_CA is not set.
 #ISTIO_CA
+
+#set CA_ADDR if your istiod is running on port other than 15012
+#CA_ADDR=istiod.istio-system.svc:32108
+# set this as true on VM since no token on VM, this will make the key/cert rotation job works
+AlwaysValidTokenFlag=true
```

Comment:
Why do we need a new flag? Can we just enable this if PROV_CERT is true?

Reply:
good idea, Let me reuse PROV_CERT since it is a VM specific flag as well.